src/read.c

```diff
@@ -524,7 +524,7 @@ static avifDecoderItem * avifMetaFindItem(avifMeta * meta, uint32_t itemID)
 typedef struct avifDecoderData
 {
     avifFileType ftyp;
-    avifRWData ftypData; // cloned from avifIO
+    avifRWData ftypData; // Contents of the ftyp box. Cloned from avifIO. ftyp.compatibleBrands points into ftypData.
```

Comment:
A more robust design might be to have ftyp.compatibleBrands own the memory that it points to. In this pull request I only document the current design.